hydrus/metadata/drone_doc_gen.py

```diff
@@ -14,15 +14,15 @@ def drone_doc(API, BASE_URL):
                        BASE_URL)
 
     # Status Class
-    status = HydraClass("Status", "Status", "Class for drone status objects")
+    status = HydraClass("http://hydrus.com/Status", "Status", "Class for drone status objects")
```

Comment:
There is no problem if the links are not dereferenced. Wanted to know why we couldn't use the actual URI, since it can be dereferenced as well. It's fine